blueocean-core-js/src/js/services/BunkerService.js

```diff
@@ -33,7 +33,11 @@ export class BunkerService {
      * @returns {any} The key for the store.
      */
     bunkerKey(data) {
+      if(data && data._links && data._links.self && data._links.self.href){
         return data._links.self.href;
+      } else {
+        return Math.random().toString(36).substring(2) + (new Date()).getTime().toString(36);
```

Comment:
So asking around, all items in the bunker should have a deterministic key. Using a random string fixes the symptom, but doesn't fix the problem.

In your BunkerService-data, you have `"_class": "io.jenkins.blueocean.rest.impl.pipeline.PipelineRunImpl",` for both cases. From your testing (cause i don't really know how to reproduce it), is it only pipelinerunimpl that don't have a self link?